.circleci/config.yml

```diff
@@ -134,6 +123,8 @@ workflows:
       - minikube_local_cluster_tests:
           pre-steps:
             - prepare_for_local_cluster_tests
+          requires:
+            - test
```

Comment:
we need to run the "test" job in orther to have in the registry the image that later is used by the integration tests...